ci/bazel-scripts/diff.sh

```diff
@@ -55,19 +50,4 @@ if [ ${#files[@]} -eq 0 ]; then
     exit 0
 fi
 
-if [[ $BAZEL_COMMAND =~ ^build[[:space:]] ]]; then
-    TARGETS=$(bazel query "rdeps(//..., set(${files[*]}))")
-elif [[ $BAZEL_COMMAND =~ ^test[[:space:]] ]]; then
-    EXCLUDED_TAGS=(manual $EXCLUDED_TEST_TAGS)
```

Comment:
If you're no longer referencing `EXCLUDED_TEST_TAGS` you should remove [its definition in `.github/workflows/ci-main.yml`](https://sourcegraph.com/github.com/dfinity/ic/-/blob/.github/workflows/ci-main.yml?L75-L76).